### DISCUSSION TITLE
Event notifications

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [Windos/BurntToast#206] [General]
I have a CIM indication event registered. The defined action writes to a log file, which works. I'd like to add a toast notification, but when it tries to run, I get an error.

```powershell
Submit-BTNotification: Exception calling ".ctor" with "1" argument(s): "The application called an interface that was marshalled for a different thread. (0x8001010E
(RPC_E_WRONG_THREAD))"
```

I am not too surprised by the error. Is there a work around?

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
